tasks/main.yml

```diff
@@ -197,9 +197,10 @@
 - name: Add destination for mailman
   lineinfile:
     dest:  "/etc/postfix/destinations"
-    line:  "{{mailman_domain}} OK"
-    regexp: "^{{mailman_domain}}"
+    line:  "{{item}} OK"
+    regexp: "^{{item}}"
     insertbefore: "BOF"
+  with_items: mailman_domains
```

Comment:
[DEPRECATION WARNING]: Using bare variables is deprecated. Update your playbooks so that the environment value uses the full variable syntax ('{{mailman_domains}}').
This
feature will be removed in a future release. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.